scss/adminLessonSideNav.module.scss

```diff
@@ -0,0 +1,104 @@
+@use 'colors.module.scss';
+
+// Not available in the design spec colors
+$light-border: hsla(0, 0%, 85%);
+
+@mixin module {
+  padding: 20px 16px;
+  border-radius: 10px;
+  cursor: pointer;
+}
+
+.container {
+  background-color: colors.$bg-light-gray;
+  padding: 28px 21px;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 460px;
+  width: 100%;
+  max-width: 369.01px;
+  row-gap: 23px;
+  justify-content: center;
+}
+
+.container__heading {
+  font-weight: bold;
+  font-size: 18px;
+  font-family: Inter;
+  font-weight: 700;
```

Comment:
Inter 700 font file doesn't exist. So the font won't display bold.

Reply:
Inter bold font has been added in #1832